docker/demo/config/test-suite/complex-dag-mor.yaml

```diff
@@ -81,11 +81,7 @@ second_hive_query:
       query1: "select count(*) from testdb.table1 group by `_row_key` having count(*) > 1"
       result1: 0
       query2: "select count(*) from testdb.table1"
-      result2: 3100
-      query3: "select count(*) from testdb.table1_rt group by `_row_key` having count(*) > 1"
-      result3: 0
-      query4: "select count(*) from testdb.table1_rt"
-      result4: 3100
+      result2: 1100
```

Comment:
while you are at it, since rollback is not effective, can we remove it from this dag file until we have a fix. I see this dag more as a sanity check dag and so everything should be succeeding.

Reply:
same for cow dag as well.